templates/openstack/consulting.html

```diff
@@ -8,158 +8,253 @@
+        <p>Hundreds of successful OpenStack clouds underpin our consulting, training, architecture design, and hardware procurement advice - to help you evaluate and deliver OpenStack.</p>
+        <p>
+          <a href="/openstack/contact-us?product=foundation-cloud" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Talk to us', 'eventLabel' : 'Talk to us - Top CTA', 'eventValue' : undefined });" class="p-button--positive">Talk to us</a>
+          &#8195;
+          <a href="{{ ASSET_SERVER_URL }}8cf6fe4f-Foundation-Cloud-Build-Openstack-data-sheet.pdf" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Read the datasheet', 'eventLabel' : 'Read the datasheet - Top CTA', 'eventValue' : undefined });" class="p-button--neutral">Read the datasheet</a>
+        </p>
       </div>
     </div>
   </section>
 
   <section class="p-strip is-bordered">
-    <div class="row p-divider u-equal-height">
-      <div class="col-4 p-divider__block">
-        <h3>Experts design and deploy your cloud</h3>
-        <p>Canonical uses your requirements to define your unique cloud architecture, which allows your staff to focus on business applications.</p>
-      </div>
-      <div class="col-4 p-divider__block">
-        <h3>Comes with everything you need</h3>
-        <p>Foundation Cloud Builds come with all the tools and technologies required to keep your cloud operating efficiently.</p>
-      </div>
-      <div class="col-4 p-divider__block">
-        <h3>24x7 support available</h3>
-        <p>Once operational, our support plans have you covered so your cloud is maintained with guaranteed updates and upgrades for new features.</p>
-      </div>
+    <div class="row">
+      <p>Building and running an OpenStack cloud requires specialised expertise and low-level Linux OS knowledge. Canonical brings the full stack - from kernel to containers to applications - along with operating experience from hundreds of OpenStack deployments around the world to get you started efficiently.</p>
+      <p>Once you are certain OpenStack is the right choice we can deliver your cloud in two weeks. Only then do you need to commit the devops talent required to run your cloud, or <a href="/openstack/managed">outsource that to Canonical</a> as well.</p>
     </div>
-  </section>
-
-  <section class="p-strip--light is-bordered">
     <div class="row">
-      <h2>Get started</h2>
+      <h2>From zero to private cloud - OpenStack delivery on rails</h2>
+      <p>With many years experience deploying, supporting, running and upgrading OpenStack clouds, Canonical offers a unique fixed-price private cloud delivery service with a proven reference architecture, called Foundation Cloud Build. Canonical will design your cloud together with your architects, and deploy OpenStack for you, on your premises.</p>
```

Comment:
This has an extra sentence at the end. Should be: "With many years experience deploying, supporting, running and upgrading OpenStack clouds, Canonical offers a unique fixed-price private cloud delivery service with a proven reference architecture, called Foundation Cloud Build."